scripts/constants.mjs

```diff
@@ -10,8 +10,10 @@ const __filename = fileURLToPath( import.meta.url );
 const __dirname = upath.dirname( __filename );
 
 export const CKEDITOR5_ROOT_PATH = upath.join( __dirname, '..' );
+export const CKEDITOR5_PACKAGES_PATH = upath.join( CKEDITOR5_ROOT_PATH, PACKAGES_DIRECTORY );
 export const CKEDITOR5_COMMERCIAL_PATH = upath.join( CKEDITOR5_ROOT_PATH, 'external', 'ckeditor5-commercial' );
-export const CKEDITOR5_PREMIUM_FEATURES_PATH = upath.join( CKEDITOR5_COMMERCIAL_PATH, PACKAGES_DIRECTORY, 'ckeditor5-premium-features' );
+export const CKEDITOR5_COMMERCIAL_PACKAGES_PATH = upath.join( CKEDITOR5_ROOT_PATH, PACKAGES_DIRECTORY );
```

Comment:
`CKEDITOR5_PACKAGES_PATH` and `CKEDITOR5_COMMERCIAL_PACKAGES_PATH` have the same value

Reply:
Fixed